### PR TITLE
Renamed variables and a parameter

### PR DESCRIPTION
**Pull request by jericahuang**
@lgolding @Rolstenhouse @kschecht

### CODE REVIEW

File: src/Sarif/Writers/ResultLogJsonWriter.cs [hunk 1]
```diff
@@ -114,23 +114,23 @@ public void WriteTool(Tool tool)
         /// after the results, as the full list of scanned files might not be known until
         /// all results have been generated.
         /// </summary>
-        /// <param name="fileDictionary">
+        /// <param name="filesDictionary">
```
Comment (lgolding) [inline src/Sarif/Writers/ResultLogJsonWriter.cs]: Huh. There was a diagnostic on this. Well... I'll come take a look.

**Comment by lgolding** [comment]
:+1:

**Comment by lgolding** [comment]
I just deleted my +1. @jericahuang, don't you still get a warning because of the underscores in those names?

**Comment by jericahuang** [comment]
@lgolding Yes, I just noticed. I pushed a new commit and had them removed. The warning is gone now.

**Comment by lgolding** [comment]
Great! :+1:

**Pull request merged**